src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs

```diff
@@ -165,7 +165,7 @@ private void WriteChoiceSchema(CodeWriter writer, EnumType schema)
                     writer.Line($"private readonly {schema.BaseType} _value;");
                     writer.Line();
 
-                    writer.WriteXmlDocumentationSummary($"Determines if two <see cref=\"{name}\"/> values are the same.");
+                    writer.WriteXmlDocumentationSummary($"Initializes a new instance of {name}.");
```

Comment:
Shall we also use `<see cref=\"{name}\"/>` here?

Reply:
reasonable and changed